csm/cli/cortxcli.py

```diff
@@ -34,6 +34,8 @@ def __str__(self):
         return f"{self.rc}: {self.message}"
 
 class Terminal:
+
+    EMPTY_PASS_FIELD = "Password field can't be empty."
```

Comment:
Why is it not working with Const?

Reply:
since these methods are called from argparser class and const is imported in main this code is throwing an error that const not found. Hence I had it checked and fixed.